src/app/features/crypto-item/crypto-item.component.ts

```diff
@@ -35,4 +54,8 @@ export class CryptoItemComponent implements OnInit {
     this.removeItem.emit(this.item);
   }
 
+  profitLoss(): number {
```

Comment:
Could we reuse this function by moving to a service?

Reply:
Most probably